packages/resource-deployment/helm-charts/storage-web-api/values.yaml

```diff
@@ -8,28 +8,40 @@
 replicaCount: 1
 
 image:
-    name: storage-web-api-func
-    repository: ''
-    pullPolicy: Always
-    tag: latest
+  name: storage-web-api-func
+  repository: ''
+  pullPolicy: Always
+  tag: latest
 
 nameOverride: ''
 fullnameOverride: ''
 
 podAnnotations:
-    release: dev
-    releaseId: ''
+  release: dev
```

Comment:
should this file be values-dev.yaml?

Reply:
no, preserving default file name helps debug when using direct helm commands on [a chart](https://helm.sh/docs/topics/charts/#the-chart-file-structure)